developer/tutorial/plugin-layouts-3.md

```diff
@@ -69,12 +69,13 @@ We also need to add our layout to the registry via our `register.js`. We are goi
 ```js
 layout: [{
   layout: "coreLayoutBeesknees",
-  workflow: "coreProductWorkflow",
+  workflow: "coreWorkflow",
```

Comment:
Are you sure that's correct? Have you tested this? I am pretty sure it was recently changed to `coreProductWorkflow`